src/main/java/com/emc/mongoose/control/RunServlet.java

```diff
@@ -0,0 +1,42 @@
+package com.emc.mongoose.control;
+
+import com.emc.mongoose.Node;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+
+/**
+ @author veronika K. on 08.11.18 */
+public class RunServlet
+	extends HttpServlet {
+
+	private static final String CONTEXT_SEP = "/";
+	private static final int STATUS_OK = 200;
+	private static final int STATUS_ERROR = 400;
+	private final Node node;
+
+	public RunServlet(final Node node) {
+		this.node = node;
+	}
+
+	@Override
+	protected void doGet(final HttpServletRequest req, final HttpServletResponse resp)
+	throws ServletException, IOException {
+		resp.addHeader("Node status", node.status().name());
+		DateTimeFormatter dtf = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm:ss");
+		resp.addHeader("Node start time", dtf.format(node.startTime()));
```

Comment:
Please read kindly:
https://www.oreilly.com/library/view/restful-web-services/9780596529260/apc.html
Any custom HTTP header should start with "X-" and application name, for example:
`X-Mongoose-Run-Start-Date`